modules/log6x-quickstart-viaq.adoc

```diff
@@ -67,22 +69,9 @@ The `ClusterRole` resource is created automatically during the Cluster Logging O
 +
 [source,terminal]
 ----
-$ oc project openshift-logging
-----
-+
-[source,terminal]
-----
-$ oc adm policy add-cluster-role-to-user collect-application-logs -z collector
-----
-+
-[source,terminal]
-----
-$ oc adm policy add-cluster-role-to-user collect-audit-logs -z collector
-----
-+
-[source,terminal]
-----
-$ oc adm policy add-cluster-role-to-user collect-infrastructure-logs -z collector
+$ oc adm policy add-cluster-role-to-user collect-application-logs -z collector -n openshift-logging
+$ oc adm policy add-cluster-role-to-user collect-audit-logs -z collector -n openshift-logging
+$ oc adm policy add-cluster-role-to-user collect-infrastructure-logs -z collector -n openshift-logging
```

Comment:
The old version (commands in a separate code blocks) is correct. 

- Consider reverting this change. See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#single-command-per-code-block

> Do not use more than one command per code block.

- Commands need to have an instruction for them. See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command